CONTRIBUTING.adoc

```diff
@@ -0,0 +1,30 @@
+[[contribution-guidelines]]
+= Contribution guidelines
+
+[[filing-issues]]
+== Filing issues
+
+File issues using the standard
+https://github.com/minishift/minishift-addons/issues[Github issue tracker] for the repository.
+Before you submit a new issue, we recommend that you search the list of issues to see if anyone already submitted a similar issue.
+
+[[contributing-patches]]
+== Contributing patches
+
+Thank you for your contributions! Please follow this process to submit a patch:
+
+.  Create an issue describing your proposed change to the repository.
+.  The repository owners will triage and respond to your issue promptly.
```

Comment:
Agree with @hferentschik lets remove `The repository owners will triage and respond to your issue promptly` as triaging is not compulsory for sending a PR